traffic_monitor/experimental/traffic_monitor/index.html

```diff
@@ -141,6 +132,7 @@
 			 setInterval(getCacheDownCount, 4832);
 			 setInterval(getVersion, 10007); // change to retry on failure, and only do on startup
 			 setInterval(getTrafficOpsUri, 10019); // change to retry on failure, and only do on startup
+			 setInterval(getTrafficOpsCdn, 10019); // change to retry on failure, and only do on startup
```

Comment:
Not a big deal, but it'd be ideal if this didn't match the getTrafficOpsUri interval, so they don't poll at exactly the same time.

Alternatively, I noticed a lot of browsers seems to bog down with so many timers, I've been wondering if it'd be better to have a single timer that does all the HTTP requests sequentially.